### ISSUE TITLE
Remove false positive hardreset.info

### ISSUE THREAD

**Issue by MrDisguised** [DandelionSprout/adfilt#883]
### Describe the problem below this line as meticulously and detailed as possible (incl. pagelinks if any)

https://www.hardreset.info is a website that makes educational content on how to root your phones , how to enter into download and recovery modes etc etc. This website is blocked in [Dandelion Sprout's Anti-Malware List](https://github.com/DandelionSprout/adfilt/tree/master/Alternate%20versions%20Anti-Malware%20List) (in all of them). Is it possible to remove that website from the list ? 

### Add screenshots below if needed

![image](https://github.com/DandelionSprout/adfilt/assets/128170542/b5d3f264-db94-497a-8dd6-cb0d3636f90e)

### Which adblocker(s) did you use when testing this?

uBlock Origin

### Adblocker version(s)

1.50.0

### Which filterlists did you use? Failing to tell this will temporarily close the report until it has been told.

💊 Dandelion Sprout's Anti-Malware List («hosts» file version)

### Which browser(s) did you use when testing this?

Firefox

### Browser version(s)

Firefox 114.0.2 (64-bit)

### Which OS(s) did you use when testing this?

Linux

### OS version(s)

OS: Arch Linux x86_64
Kernel: 6.3.9-zen1-1-zen

**Comment by DandelionSprout** [comment]
There was a point in time where that site (and many other root guide websites) recommended using `fastboot flash recovery twrp.img` for ***all*** phones in their copypasted guides, even ones with A/B partitions, which'd make the guides **actively dangerous**. In the case of HardReset, this was most likely 2020-2021.

While the guides still seem hard to follow when I check them tonight, they no longer seem to pose physical risks to phones, so I'll unblock the site within 24 hours.
